lib/gui/utils/byte-size/byte-size.js

```diff
@@ -26,6 +26,11 @@
 const angular = require('angular');
 const MODULE_NAME = 'Etcher.Utils.ByteSize';
 const ByteSize = angular.module(MODULE_NAME, []);
+
+/* eslint-disable lodash/prefer-lodash-method */
```

Comment:
This is happening 4 times across the code base:

- 2 of them because of the Angular way to declare filters (e.g: `module.filter`)
- 1 of them because of the RxJS `.map` function
- 1 of them because of the ImmutableJS `.find` function

I rather deal with these 4 cases than risking some inconsistencies because of not catching something during a code review. I do have some ideas to make this better though:

- We only use ImmutableJS stuff in the store, to I guess we can disable the rule for that whole file.
- We'll move from Angular soon, which means half of the annotations will disappear. Angular filters are just some syntax sugar, so we could even not make filters and we should be OK

What do you think?